openrr-apps/src/utils.rs

```diff
@@ -126,7 +137,119 @@ pub fn init_with_anonymize(name: &str, config: &RobotConfig) {
 
 pub fn init_tracing() {
     tracing_subscriber::fmt()
-        .with_env_filter(tracing_subscriber::EnvFilter::from_default_env())
+        .with_env_filter(EnvFilter::from_default_env())
         .with_writer(io::stderr)
         .init();
 }
+
```

Comment:
Could you add some simple tests for these?
(Simple means that you don't have to think of coverage rate.)
The tests will be run on windows too.